.travis.yml

```diff
@@ -27,19 +27,22 @@ matrix:
       python: '3.7'
       install:
         - pip install -U pip setuptools "mypy~=0.740"
+        - pip install -U importlib-metadata~=5.0 # this needs to be installed separately BEFORE trying to parse the setup.py file
```

Comment:
This line is here again for the same reason as my pymetrics PR. For some reason python 3.7 needs to install an updated version of importlib-metadata BEFORE it can actually do anything with the setup.py file.